src/core/congestion_control.h

```diff
@@ -13,10 +14,34 @@ typedef struct QUIC_ACK_EVENT {
 
     uint64_t LargestPacketNumberAcked;
 
+    uint64_t LargestPacketNumberSent;
+
+    //
+    // Number of retransmittable bytes acked during the connection's lifetime
+    //
+    uint64_t NumTotalAckedRetransmittableBytes;
+
+    QUIC_SENT_PACKET_METADATA* AckedPackets;
+
     uint32_t NumRetransmittableBytes;
 
     uint32_t SmoothedRtt;
 
+    uint32_t MinRttSample;
```

Comment:
I believe we already track this somewhere else. Do we really need to duplicate that?

Reply:
I'm afraid not.

This is tracking the smallest RTT we gathered from ack blocks from processing 1 ack frame.

And yes, we have tracked the smallest RTT in `QuicConnUpdateRtt`. But it for the global min/max RTT for the path, but not the local minima.

Reply:
This field has been removed, so I'm sort of forced to assume you changed your mind and agree that the field wasn't needed. So I guess I'll resolve the comment?